share/spack/gitlab/pr_pipeline.yml

```diff
@@ -17,4 +17,15 @@ merge_pipeline:
     SPACK_REF: develop
   trigger:
     project: spack/e4s
-    strategy: depend
\ No newline at end of file
+    strategy: depend
+
+release_pipeline:
+  only:
+  - releases/v0.15
+  variables:
+    SPACK_REPO: https://github.com/spack/spack.git
+    SPACK_REF: releases/v0.15
+  trigger:
+    project: spack/e4s
+    branch: releases/v0.15
```

Comment:
@alalazo I think we may have made the `releases/v01.5` branch on `gitlab.spack.io/spack/e4s` for some good reason, but we probably haven't given enough thought yet to what we should be doing with it.  I see it has an appropriately named cdash build group, but I wonder if we should also put the release binaries in a separate mirror within the S3 bucket?  In addition, it looks like the branch needs a couple updates to pkgs so that it will build properly.  I'm thinking maybe that branch should contain the same set of packages as we test for `develop`.

@tgamblin Any thoughts on the matter?  I think we should figure this stuff out and then I'll update that branch, before this can be merged.